common/src/main/java/com/wynntils/features/TerritoryDefenseMessageFeature.java

```diff
@@ -37,10 +49,32 @@ public void onInventoryClick(InventoryMouseClickedEvent event) {
             Matcher matcher = StyledText.fromComponent(tooltipLine)
                     .getMatcher(TERRITORY_DEFENSE_PATTERN, PartStyle.StyleType.NONE);
             if (matcher.matches()) {
-                Handlers.Command.sendCommandImmediately(
-                        "g %s defense is %s".formatted(titleMatcher.group(1), matcher.group(1)));
+                // intentionally not localized to match Wynncraft language
+                queuedTerritories.add(new QueuedTerritory(
+                        titleMatcher.group(1),
+                        System.currentTimeMillis(),
+                        GuildResourceValues.fromString(matcher.group(1))));
                 return;
             }
         }
     }
+
+    @SubscribeEvent(priority = EventPriority.HIGHEST)
+    public void onAttackSound(SoundPlayedEvent event) {
+        if (queuedTerritories.isEmpty()
+                || event == null
+                || !event.getSoundInstance().getLocation().equals(ATTACK_SOUND)) return;
```

Comment:
```suggestion
        if (queuedTerritories.isEmpty()
                || !event.getSoundInstance().getLocation().equals(ATTACK_SOUND)) {
                return;
        }
```

event can't be null

Reply:
Did you mean event.getSoundInstance() == null?

Reply:
No, it was just left over when I was testing the sound mixin.